lib/bamboo/phoenix.ex

```diff
@@ -122,7 +122,7 @@ defmodule Bamboo.Phoenix do
 
       def text_email_layout do
         new_email
-        # Will use MyApp.LayoutView with email.text layout when rendering text emails
+        # Will use MyApp.LayoutView with email.text template when rendering text emails
```

Comment:
Hmm I keep going back and forth on this one. In the method definitions though we have
`put_text_layout(email, layout)`, so right now in a variety of places we're referring to that `email.text` or `email.html` as a layout. So, idk. I'd be fine with referring to it as a template but if so I think we need to change it everywhere.